WordPress/Helpers/SanitizationHelperTrait.php

```diff
@@ -236,4 +252,166 @@ public function is_sanitizing_function( $functionName ) {
 	public function is_sanitizing_and_unslashing_function( $functionName ) {
 		return isset( $this->get_sanitizing_and_unslashing_functions()[ strtolower( $functionName ) ] );
 	}
+
+	/**
+	 * Check if something is only being sanitized.
+	 *
+	 * @since 0.5.0
+	 * @since 3.0.0 - Moved from the Sniff class to this class.
+	 *              - The method visibility was changed from `protected` to `public`.
+	 *              - The `$phpcsFile` parameter was added.
+	 *
+	 * @param \PHP_CodeSniffer\Files\File $phpcsFile The file being scanned.
```

Comment:
This is probably something we can address in a separate PR, but since we have the import of this class at the top, I don't think we need to write the entire namespace in the docblock (unless we really want to be explicit, but we can discuss this in an issue maybe).

Just leaving a comment here not to forget about it later on 😄

Reply:
Opened the issue: https://github.com/WordPress/WordPress-Coding-Standards/issues/2358

Reply:
`use` statements have no effect on docblocks, so it's a matter of preference. Currently we are using FQNs everywhere and it saves having to scroll up in the file to check where something is coming from.